helpers/dom.js

```diff
@@ -1,3 +1,11 @@
+export function cssEscape(val) {
+	if (window.CSS && window.CSS.escape) {
+		return window.CSS.escape(val);
+	}
+	val = val.replace(/\$/g, '\\$');
```

Comment:
There's a [real polyfill](https://github.com/mathiasbynens/CSS.escape) for this, but since this is really just a temporary thing unti legacy-Edge is gone in a couple months, I think we're good with a specific fix for dollar signs.